website/content/en/docs/main/getting-started/users.md

```diff
@@ -25,6 +25,16 @@ For example, you can set up a local environment using [kind][].
    clusterlink --version
    ```
 
+{{% expand summary="Download specific CLI version" %}}
+   To install a specific version of the ClusterLink CLI, add the `$VERSION` flag to the installation command.
+   For example, to download version v0.2.1:
+
+   ```sh
+   curl -L https://github.com/clusterlink-net/clusterlink/releases/latest/download/clusterlink.sh | VERSION=v0.2.1 sh -
```

Comment:
isn't there a specific download URL for each version?
Or is the download done by the `clusterlink.sh` script?